docs/src/00-doc/05-decisions-and-ADRs/adr/0009-rtl-support.stories.mdx

```diff
@@ -24,8 +24,25 @@ We identified two challenges:
 
 Fortunately, both problems have solutions in the form of postcss plugins ([postcss-logical](https://www.npmjs.com/package/postcss-logical), [postcss-dir-pseudo-class](https://www.npmjs.com/package/postcss-dir-pseudo-class)) and a stylelint plugin ([stylelint-use-logical](https://www.npmjs.com/package/stylelint-use-logical)).
 
+### RTL support for icons
+Neither of the tools and solutions discussed above concern themselves with icons.
+
+#### Symmetrical icons
+Symmetrical icons don’t need to be flipped between LTR and RTL layouts.
+
+#### Non-symmetrical icons
+Non-symmetrical icons can be flipped (with css transform, e.g. [data-bridge button](https://gerrit.wikimedia.org/r/plugins/gitiles/mediawiki/extensions/Wikibase/+/refs/heads/master/client/data-bridge/src/presentation/components/EventEmittingButton.vue#289)) for RTL scripts.  
+They can also be provided with two separate `svg`s - one for LTR and one for RTL. The correct `svg` should be loaded based on the `dir` attribute.  
```

Comment:
Even after doing some research, I'm not sure of which one of the two approaches would perform better, but – as metioned in Mattermost – the system iconography will contain RTL versions of the non-symmetrical icons that need adjustment, so the "separate svg" approach would be covered.